### PR TITLE
Fix #1959 Print version info

### LINKED ISSUE #1959: Print version info
We'll need a way to print the version info using the `peer` binary.

The expected behavior is that `peer version`, `peer -v`, and `peer --version` should return version info by reading the `peer.version` key in the `core.yaml` file and quit.

This is a fairly straightforward task and a good opportunity for someone to get their hands wet with submitting patches to the project. They'll be using the [cobra](https://github.com/spf13/cobra), [viper](https://github.com/spf13/viper), and [pflag](https://github.com/ogier/pflag) packages.

@srderson If you agree, can we add the "Help wanted" tag to invite participation?

### PR DESCRIPTION
**Pull request by GrapeBaBa**
## Description

Add version command and --version/-v flag for peer service.
## Motivation and Context

Fixes #1959 
## How Has This Been Tested?

make checks successful
## Checklist:

- [x] I have added a [Signed-off-by](https://github.com/hyperledger/fabric/blob/master/CONTRIBUTING.md#legal-stuff).
- [x] I have either added documentation to cover my changes or this change requires no new documentation.
- [x] I have either added unit tests to cover my changes or this change requires no new tests.
- [x] I have run [golint](https://github.com/golang/lint) and have fixed valid warnings in code I have added or modified. This tool generates false positives so you may choose to ignore some warnings. The goal is clean, consistent, and readable code.

Signed-off-by:Kai Chen ckaiwh@cn.ibm.com

### CODE REVIEW

**Comment by MacusMeng** [comment]
Good job!

**Comment by srderson** [comment]
@GrapeBaBa Nice! Could you update the documentation at https://github.com/hyperledger/fabric/blob/master/docs/API/CoreAPI.md#cli to include this command?

**Comment by GrapeBaBa** [comment]
@srderson let me do it.

**Comment by GrapeBaBa** [comment]
reverify jenkins

**Comment by GrapeBaBa** [comment]
anyone can re-trigger the travis build?

**Comment by srderson** [comment]
I restarted Travis

File: docs/API/CoreAPI.md [hunk 1]
```diff
@@ -35,20 +37,23 @@ You will see output similar to the example below (**NOTE:** rootcommand below is
...
-The `peer` command supports several subcommands, as shown above. To
+The `peer` command supports several subcommands and flags, as shown above. To
 facilitate its use in scripted applications, the `peer` command always
 produces a non-zero return code in the event of command failure. Upon success,
 many of the subcommands produce a result on **stdout** as shown in the table
 below:
 
 Command | **stdout** result in the event of success
 --- | ---
+`peer version`     | String form of `peer.version` defined in [core.yaml](https://github.com/hyperledger/fabric/blob/master/peer/core.yaml)
```
Comment (srderson) [inline docs/API/CoreAPI.md]: "`peer version`     | String form of `peer.version` defined in [core.yaml]" should be "`version`     | String form of `peer.version` defined in [core.yaml]"

**Comment by srderson** [comment]
One minor comment on the documentation, otherwise LGTM

It seems we have no testing around the commands/flags. We should probably open an issue for that to address in a separate PR. Thoughts?

**Comment by srderson** [comment]
@GrapeBaBa Thanks! LGTM

**Pull request merged**